build.cmd

```diff
@@ -18,7 +18,9 @@ if %errorlevel% equ 0 (
 )
 
 if "%outloop%" equ "true" (
+        pushd src\System.Private.ServiceModel\tools\setupfiles
```

Comment:
Alternative to pushd/popd is to use the /D option to pass in the working directory:
    start /D src\System.Private.ServiceModel\tools\setupfiles /wait BuildWCFTestService.cmd